src/pages/dmt5/_margin_calculator.js

```diff
@@ -0,0 +1,292 @@
+        width: 100%;
+        margin: 0 0 24px;
+        height: unset;
+
+        div {
+            max-width: 232px;
+            width: 100%;
+        }
+    }
+`
+
+const MainHeader = styled(Header)`
+    font-size: 48px;
+    margin: 0 0 12px;
+    line-height: 1.25;
+    text-align: left;
+    color: var(--color-black-3);
+    @media ${device.mobileL} {
+        font-size: 32px;
+        margin-bottom: 24px;
+        padding: 0 16px;
+        text-align: center;
+    }
+`
+
+const SubHeader = styled(Header)`
+    font-size: 24px;
+    color: var(--color-black-3);
+    @media ${device.mobileL} {
+        font-size: 20px;
+        margin-bottom: 8px;
+    }
+`
+
+const StyledBox = styled(Box)`
+    max-width: 508px;
+    margin: 11.9rem 4rem 0 16rem;
+    @media ${device.mobileL} {
+        max-width: 100%;
+        margin: 4rem 1.6rem 2.4rem 1.6rem;
+    }
+`
+
+const StyledText = styled(Text)`
+    text-align: ${(props) => (props.align ? 'center' : 'left')};
+    margin-bottom: 24px;
+    @media ${device.mobileL} {
+        font-size: 16px;
+    }
+`
+
+const CardText = styled(StyledText)`
+    margin: 0.8rem 4rem 2.4rem 4rem;
+    @media ${device.mobileL} {
+        margin: 0 0 24px 0;
+    }
+`
+
+const StyledLinkButton = styled(LinkButton)`
+    padding: 1rem 1.6rem;
+    font-size: 14px;
+    height: 4rem;
+    width: 21rem;
+    @media ${device.mobileL} {
+        padding: 1rem 1.5rem;
+        width: 198px;
+    }
+
+    :active {
+        outline: none;
+        border: none;
+    }
+    :focus {
+        outline: 0;
+    }
+`
+
+const StyledFlexContainer = styled(Flex)`
+    border: 1px solid var(--color-grey-33);
+`
+
+const StyledFlexRight = styled(Flex)`
+    margin-right: 2.4rem;
+    @media ${device.mobileL} {
+        margin: 0;
+    }
+`
+
+const StyledFlexLeft = styled(Flex)`
+    background-color: var(--color-grey-25);
+    margin: 0;
+`
+
+const CalculatorCard = ({ name, image, image_alt_name, text, link, button_text, is_mobile }) => (
+    <Flex
+        jc="center"
+        tablet_jc="center"
+        ai="center"
+        fd="column"
+        wrap={is_mobile ? 'nowrap' : 'wrap'}
+        width={is_mobile ? '100%' : '58.8rem'}
+        height={is_mobile ? 'auto' : '69.4rem'}
+    >
+        <SubHeader align="center">{name}</SubHeader>
+        <CardText align="center">{text}</CardText>
+        <ImageWrapper>
+            <QueryImage data={image} alt={image_alt_name} />
+        </ImageWrapper>
+        <StyledLinkButton tertiary="true" to={link} external="true" target="_blank">
+            <Localize translate_text={button_text} />
+        </StyledLinkButton>
+    </Flex>
+)
+
+CalculatorCard.propTypes = {
+    button_text: Proptypes.string,
+    image: Proptypes.object,
+    image_alt_name: Proptypes.string,
+    image_name: Proptypes.string,
+    is_mobile: Proptypes.bool,
+    link: Proptypes.string,
+    name: Proptypes.string,
+    text: Proptypes.string,
+}
+
+const calculators = [
+    {
+        index: 0,
```

Comment:
Why do we need index here?

Reply:
The index is used to map the calculator data to the carousel. It is important for child components to have a unique key: https://reactjs.org/docs/lists-and-keys.html